utils/kafka-test-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaClient.scala

```diff
@@ -76,18 +151,9 @@ class KafkaClient(kafkaAddress: String, id: String) extends LazyLogging {
     adminClient.close()
   }
 
-  def createConsumer(groupIdOpt: Option[String] = None): KafkaConsumer[Array[Byte], Array[Byte]] = synchronized {
```

Comment:
Why it is better to have consume* methods inlined in kafka client instead of creating consumer and then invoking consume* methods on it? We are using kafka very low level, so IMO we should stick to abstracts given by kafka. What about situation when you want to test additional things like rebalance, sending commits, explicitly closing consumer, etc?